packages/composites/apy-finance/test-payload.js

```diff
@@ -0,0 +1,40 @@
+const basePayload = {
+  request: {},
+}
+
+const testAndReplace = [
+  ['AMBERDATA_DATA_PROVIDER_URL', 'amberdata'],
```

Comment:
I would probably make these objects so that if nops need to make their own payloads, they'll have an easier time creating them
```js
const environmentVariables = [{envKey: "AMBERDATA_DATA_PROVIDER_URL", defaultValue: "amberdata" }]
```